tests/runtest.proj

```diff
@@ -354,7 +352,7 @@ namespace $([System.String]::Copy($(Category)).Replace(".","_").Replace("\","").
   <!-- All the test projects need to add dependency to currently built runtime as they require that to run. 
        In addition the version number of built runtime can change so all project.json needs to be dynamically generated. 
        In order to avoid regenerating project.json for all the tests projects...I added a common project for all the tests.
-       This project is at tests\src\common\test_runtime\test_runtime.csproj. This project is adding common dependencies to
+       This project is at tests\src\common\test_dependencies\test_dependencies.csproj. This project is adding common dependencies to
        all the tests like dependency on Microsoft.netcore.runtime.coreclr ..etc. Project.json for this project is dynamically 
```

Comment:
Yes you are right, the file referred here has not been used for the above purpose for a long time. will remove the file from the comment